scrapers/scrape_bs.py

```diff
@@ -1,8 +1,28 @@
 #!/usr/bin/env python3
+import requests
+from bs4 import BeautifulSoup
+
 
 import scrape_common as sc
+import utils.bs as bs
+
+def text_from_url(url: str) -> str:
+    req = requests.get(url)
```

Comment:
Please add this:

```
  print(f'Downloading: {url}')
```

just before `requests.get`, so we capture it in parser and CSV files.